DarkDream99/AlgoBridge-web
# Feature/ab code ide stack

**DESCRIPTION**:

Add base editor stack for algorithm development:

- create variable and available to assign some value;
- base operation: +, -, *, /;
- logic operation: and, or;
- construction: if, for;
- condition operations: >, >=, <, <=, =;
- functions;

Add code IDE that consist of operation rows (user can move the row up or down, create new one, or delete selected row).


**HOW TO TEST:**

1. Pull this branch.
2. Go to url: `your-host`/algo/new.
3. Check that operation editor is opened correctly when you click on the operation body.
4. Check that code IDE works properly.
